### PR TITLE
Prepare dockerfile

### PR DESCRIPTION
**Pull request by alszczep**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+ARG NODE_VERSION
```
Comment (jakubfijalkowski) [inline Dockerfile:2]: Don't. Parametrizing `Dockerfile` base images is a bad idea. It tends to get out of hand really quickly and you just don't really know what the image is based upon in a particular point in time.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (jakubfijalkowski) [inline Dockerfile:1]: Unnecesary empty line.

Comment (jakubfijalkowski) [reply Dockerfile:1]: Also, this image basically does nothing. Is this expected (temporarily)?

**Review by konowrockis — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (konowrockis) [reply Dockerfile:1]: This is pretty much final version of this image. Everything required for the actual app will be inside the app repo. This is only bnanse for setting envs and adding necessary chromium bloatware for puppeteer.

File: Dockerfile (hunk 1 above)
Comment (konowrockis) [reply Dockerfile:2]: Ah, I forgot, I think we wanted to add base node version to tags. So basically something like
`:1.2.3-node-17, :1.2.3, :latest` Would that be ok?

Comment (jakubfijalkowski) [reply Dockerfile:2]: Okay, but if you use different base images - have different dockerfiles. There will be differences in the necessary dependencies of you target different OSes (or different version of OS, and base image will eventually do that).

File: Dockerfile (hunk 2 above)
Comment (alszczep) [reply Dockerfile:1]: It is. If I understood correctly, this is just some kind of template containing what is needed for pdf-renderer package to work. Does this change anything about parametrization?

**Review by jakubfijalkowski — APPROVED** [review]

**Pull request merged**